css/styles.css

```diff
@@ -0,0 +1,855 @@
+
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+  overflow-x: hidden;
+}
+
+.social-links {
+  display: none;
+}
+
+.navigation {
+  margin-left: 30px;
+}
+
+.nav-icons {
+  font-size: 30px;
+}
+
+.logo-image {
+  display: none;
+}
+
+#bar {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  width: 48px;
+  height: 48px;
+  color: #272a31;
+  margin-left: 30px;
+}
+
+.close-menu a {
+  margin: 30px;
+  color: #272a31;
+}
+
+.mobile-links {
+  margin-bottom: 20px;
+}
+
+.nav-menu li {
+  list-style: none;
+  display: flex;
+}
+
+.nav-about {
+  color: #ec5242;
+}
+
+.btn-join {
+  padding-bottom: 40px;
+  padding-left: 40px;
+  padding-right: 40px;
+}
+
+.nav-menu a:hover {
+  color: #ec5242;
+  border: 1px solid #272a31;
+}
+
+.desktop-nav {
+  display: none;
+}
+
+.campain {
+  color: #ec5242 !important;
+  border: 2px solid #ec5242;
+  padding: 10px;
+}
+
+.info-head {
+  width: 100%;
+  background-image: url(../images/mobile_intro_bg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  height: auto;
+}
+
+h1 {
+  font-size: 35px;
+  font-weight: 900;
+  color: #ec5242;
+  margin: 40px 0;
+  margin-left: 20px;
+}
+
+.hello-text {
+  width: inherit;
```

Comment:
- [ ] In this case, `width: inherit;` is being a reason of whitspace on the right side of the document on mobile width, I suggest to delete it:

https://user-images.githubusercontent.com/77038610/163408319-a1e9bdac-80d9-4d8f-9645-0930c1fddb4a.mp4

Reply:
Now fixed 👍🏽